### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by saeedmb8**

### CODE REVIEW

File: docs/getting-started.rst [hunk 1]
```diff
@@ -90,8 +90,8 @@ some amount of customization. Use the ``unpack-templates`` command in the
 scaffolded application to unpack a package's templates in your application's
 templates directory::
 
-   $ molo scaffold testapp \
-   >   --require=molo.profiles \
```
Comment (miltontony) [inline docs/getting-started.rst]: These are still needed

**Comment by saeedmb8** [comment]
@milton ready for review :)

File: README.rst [hunk 2]
```diff
@@ -1,9 +1,193 @@
...
+more libraries that are required for installation::
+
+   $ molo scaffold myapp --require=django-contrib-comments
+
+Adds the `django-contrib-comments` to the generated requirements file which
+is read by the generated package's ``setup.py`` file.
+
+Multiple requires can be specified on the command line::
+
+   $ molo scaffold myapp --require=django-contrib-comments
```
Comment (smn) [inline README.rst]: missing `\`

File: README.rst [hunk 3]
```diff
@@ -1,9 +1,193 @@
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Sometimes a package's existing templates simply are not enough and need
+some amount of customization. Use the ``unpack-templates`` command in the
+scaffolded application to unpack a package's templates in your application's
+templates directory::
+
+   $ molo scaffold testapp
+   >   --require=molo.profiles
+   >   --include=molo.profiles ^profiles/
```
Comment (smn) [inline README.rst]: missing `\`s

**Comment by saeedmb8** [comment]
@smn @miltontony ready for review

**Comment by smn** [comment]
:+1: :cake: :)

**Pull request merged**